spec/numbers_and_words/integer/fixture_examples/pt-BR.yml

```diff
@@ -0,0 +1,134 @@
+to_words:
+  ones:
+    0: zero
+    1: um
+    9: nove
+  teens:
+    10: dez
+    11: onze
+    19: dezenove
+    20: vinte
+    21: vinte e um
+    80: oitenta
+    90: noventa
+    99: noventa e nove
+  hundreds:
+    100: cem
+    101: cento e um
+    111: cento e onze
+    120: cento e vinte
+    121: cento e vinte e um
+    151: cento e cinquenta e um
+    800: oitocentos
+    900: novecentos
+    909: novecentos e nove
+    919: novecentos e dezenove
+    990: novecentos e noventa
+    999: novecentos e noventa e nove
+  thousands:
+    1000: mil
+    1800: mil e oitocentos
+    1880: mil oitocentos e oitenta
+    1881: mil oitocentos e oitenta e um
+    2000: dois mil
+    2800: dois mil e oitocentos
+    2880: dois mil oitocentos e oitenta
+    2801: dois mil oitocentos e um
+    2017: dois mil e dezessete
+    2045: dois mil e quarenta e cinco
+    4000: quatro mil
+    5000: cinco mil
+    5002: cinco mil e dois
+    11000: onze mil
+    21000: vinte e um mil
+    21001: vinte e um mil e um
+    999000: novecentos e noventa e nove mil
+    999999: novecentos e noventa e nove mil e novecentos e noventa e nove
```

Comment:
Could you verify this is correct? I think it should be how it actually is (if so, other numbers should change as well):

```yml
999999: novecentos e noventa e nove mil novecentos e noventa e nove
```

Looking at a few documents:

https://www.legisweb.com.br/legislacao/?id=178282 Art. 3º, Seção 2º

> ... o número nove milhões, novecentos e noventa e nove mil novecentos e noventa e nove ...

Also:
 
http://www.languagesandnumbers.com/how-to-count-in-portuguese-brazil/en/por-bra/